src/share.scss

```diff
@@ -7,4 +7,4 @@ $gray1: #f4f4f4;
 $gray2: #ddd;
 $gray3: #ccc;
 $gray4: #5b6770;
-$gray5: #333;
+$gray5: #333;
```

Comment:
If you run "npm run format", it will format all the .js and .scss files.
It always adds a blank line to all the files.